src/lib/network_pool/transaction_pool.ml

```diff
@@ -431,7 +434,6 @@ struct
       ; mutable best_tip_ledger : (Base_ledger.t[@sexp.opaque]) Option.t
       ; verification_key_table : (Vk_refcount_table.t[@sexp.opaque])
       }
-    [@@deriving sexp_of]
 
```

Comment:
If you're removing `sexp_of` here, you should also remove the `sexp` annotations in the type above